addons/sourcemod/scripting/shop/functions.sp

```diff
@@ -284,6 +291,49 @@ public int Functions_Menu_Handler(Menu menu, MenuAction action, int param1, int
 	}
 }
 
+public void Function_ConfirmLuckMenu(int client)
+{
+	char buffer[256];
+	Menu menu = new Menu(Menu_ConfirmTryLuck);
```

Comment:
why for confirm buy and sel you use panel, but for confirm try luck you use menu?

Reply:
Because Confirm Buy menu and Confirm Sell menu has a panel function to show the item info. while confirm luck really doesn't has complex info like item, just amount of credits or maybe "chance" that client will get, so I use menu instead.